core/pbcc_actions_api.c

```diff
@@ -17,31 +17,16 @@
 
 /* Maximum number of actions to return in response */
 #define MAX_ACTIONS_LIMIT 100
+#define MAX_ACTION_VALUE_LENGTH 15
```

Comment:
```suggestion
#define MAX_ACTION_TYPE_LENGTH 15
```
Should it be like this?

Reply:
tbh, we should - commiting